README-comments.md

```diff
@@ -0,0 +1,268 @@
+reactive-sandbox   1         1         1            1           1m
+```
+
+Now clone the Chirper sample referred to by the Lightbend Orchestration for Kubernetes project:
+
+```
+git clone https://github.com/longshorej/lagom-java-chirper-tooling-example.git chirper
+cd chirper
+```
+
+Add a namespace to the Container Registry to hold the images the Chirper build process will create:
+
+```
+bx plugin install container-registry -r Bluemix
+bx cr namespace-add chirper
+```
+
+If you were using Minikube you would invoke the 'sbt-native-packager' to create Docker images using `sbt docker:publishLocal`, but we're going to switch to publish to the Container Registry. For that we need to give `sbt` the registry location and namespace. You may already know the hostname of the Container Registry - there is one in each IBM Cloud region. It's the 'Container Registry' line output when you execute `bx cr info`.
+
+For the namespace, we're going to use 'chirper'.
+
+Add the following to the `build.sbt` file to the end of the definition of the `project` method around line 122:
+
+```
+  .settings(
+    dockerRepository := Some("registry.eu-gb.bluemix.net/chirper")
+  )
+```
+
+Note: I'm in the UK so my registry hostname has `eu-gb` in its name. Yours may differ.
+
+Now, when you execute:
+
+```
+$ bx cr login
+$ sbt docker:publish
+```
+
+the first command will redirect the local Docker engine to the Container Registry, and the second command will build the code and publish the images there.
+
+You can check the images have uploaded by viewing the [IBM Cloud console page your private registry](https://console.bluemix.net/containers-kubernetes/registry/private). Which should look similar to:
+
+![Container Registry after publishing images](images/ContainerReg.png)
+
+### Deploy to Kubernetes
+Now we're ready to direct the Container Service (Kubernetes) to deploy the images and run the application.
+
+You will need to run the following to set up your environment for the deploy. Be sure to adjust the secret variables, cassandra service address and Container Registry host as necessary.
+
+```
+REGISTRY=registry.eu-gb.bluemix.net
+NAMEPSACE=chirper
+
+# Be sure to change these secret values
+
+chirp_secret="youmustchangeme"
+friend_secret="youmustchangeme"
+activity_stream_secret="youmustchangeme"
+front_end_secret="youmustchangeme"
+
+# Default address for reactive-sandbox, change if using external Cassandra
+
+cassandra_svc="_cql._tcp.reactive-sandbox-cassandra.default.svc.cluster.local"
+
+# Configure the services to allow requests to the Kubernetes IP (Play's Allowed Hosts Filter) For development purposes we'll allow all IPs.
+
+allowed_host=.
+```
+
+There are four images necessary (ignoring the load test image). For each image we'll use the Lightbend `rp` tool to generate the Kubernetes deployment YAML configuration and pipe that to the `kubectl` command to apply the configuration to Kubernetes.
```

Comment:
Maybe add another paragraph explaing that chirp-impl and friend-impl use Akka Cluster, thus needing 2 (or more) replicas so that they can properly form the cluster. Possibly link to [Akka Cluster Bootstrap](https://developer.lightbend.com/docs/akka-management/current/bootstrap.html) as well?

Reply:
OK, I agree about the 2 replicase thing.

Are you saying that Akka Cluster Bootstrap is being used in this example. I can see it's being generated into `friend-impl_2.12-1.0.0-SNAPSHOT.pom` but can't see what triggers that.